docs/content/documentation/content/page.md

```diff
@@ -74,10 +74,10 @@ The file `content/blog/2018-10-10-hello-world.md` will yield a page at `[base_ur
 
 ## Front matter
 
-The TOML front matter is a set of metadata embedded in a file at the beginning of the file enclosed
-by triple pluses (`+++`).
+The front matter is a set of metadata embedded in a file at the beginning of the file enclosed
+by triple pluses (`+++`) if in TOML format or by triple minuses (`---`) if in YAML format.
```

Comment:
Let's not document that for now, I would rather this be an undocumented feature.

Reply:
So I kept the commit about documentation but reworded it with the spin @rcarmo had in mind. Hopefully that's fine now.